c++/src/kj/table-test.c++

```diff
@@ -1317,6 +1317,94 @@ KJ_TEST("insertion order index is movable") {
   KJ_EXPECT(iter == range.end());
 }
 
+// =======================================================================================
+// Test bug where insertion failure on a later index in the table would not be rolled back
+// correctly if a previous index was TreeIndex.
+
+class StringLengthCompare {
+  // Considers two strings equal if they have the same length.
+public:
+  inline const size_t keyForRow(StringPtr entry) const {
```

Comment:
The `const` in this return type is causing the builds to fail.
```
src/kj/table-test.c++:1327:10: error: 'const' type qualifier on return type has no effect [-Werror,-Wignored-qualifiers]
  inline const size_t keyForRow(StringPtr entry) const {
         ^~~~~~
```

Reply:
Copy/pasto. Removed (no other change).